.golangci.yml

```diff
@@ -1,29 +1,267 @@
 run:
```

Comment:
Why not use the bundled config?

Reply:
I assumed the config in the Sage repo would be more up-to-date. Is that not the case?

Reply:
The config in the Sage repo is `go:embeded` and automatically used by the `sggolangcilint.Run` helper, so you don't need to bring it unless you want to customize it (not recommended)

Reply:
Or maybe you meant why not remove the file completely? That is definitively better